sklearn/multioutput.py

```diff
@@ -101,11 +101,9 @@ def partial_fit(self, X, y, classes=None, sample_weight=None):
         -------
         self : object
         """
-        X, y = check_X_y(X, y,
-                         force_all_finite=False,
-                         multi_output=True,
-                         accept_sparse=True)
-
+        X, y = self._validate_data(X, y, force_all_finite=False,
+                                   multi_output=True,
+                                   accept_sparse=True)
```

Comment:
Since this is `partial_fit`, we need to explicitly set `reset`:

```python
first_time = not hasattr(self, 'estimators_')
X, y = self._validate_data(X, ..., reset=first_time)
```